extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -408,17 +392,7 @@ protected function setup() {
 						$repeating = (
 							count( $nameTokens ) == 2 &&
 							$nameTokens[0] === self::RENAME_TAG &&
-							$nameTokens[1] === $newuser->getName()
-						);
-					}
-
```

Comment:
The diff is a little funny here;  I really just removed the whole ```if``` block below.  Maybe the newuser -> newUser rename messed it up.  First I got rid of the ```$processing = ...``` section because that variable isn't used anywhere. Once I did that I realized ```$statusTokens``` was no longer used anywhere.  Thus the whole block became irrelevant.